packages/blockchain-wallet-v4-frontend/src/modals/Btc/RequestBtc/FirstStep/selectors.js

```diff
@@ -32,7 +32,7 @@ const extractAccountIdx = value =>
         : Remote.of(value.xpub)
     : Remote.NotAsked
 
-export const getData = state => {
+export const getData = (state) => {
```

Comment:
Unnecessary change

Reply:
I think it could be prettify-induced:
my version of vscode prettify force-removes all brackets around single arguments
since there's no eslint config change this commit must've got around verification somehow